### PR TITLE
Adding Giscus Support

### PR DESCRIPTION
**Pull request by rterakedis**
Adding support for https://giscus.app/ commenting. Comments and Reactions are added and tracked in GitHub discussions. This commenting system works similarly to Utterances.

## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bug-fix
- [X] Feature (functionality, design, translations, etc.)
- [ ] Documentation change
- [ ] Project management (tests, CI, GitHub configuration, etc.)
- [ ] Other (please describe):

## Current state

Issue Number(s):  No current issue requesting this

## Proposed changes

Adding support for https://giscus.app/ commenting. Comments and Reactions are added and tracked in GitHub discussions. This commenting system works similarly to Utterances.

Giscus is loaded in the comments.html partial by referring to the giscus.html partial. Since single.html layout loads comments if comments are enabled, you must ensure both the comments and giscus parameters are configured in Site Params.

## Screenshots, if applicable

image

## Checklist

- [X] **Bug-fixes and new features:** I have tested locally with the [latest release of Hugo extended](https://github.com/gohugoio/hugo/releases). This requirement is [a standard](https://github.com/gohugoio/hugoThemes#theme-maintenance).
- [X] **Bug-fixes, new features, and doc changes:** I have updated the relevant documentation as part of this PR.
- [X] **All PRs:** I have [signed off](https://github.com/chipzoller/hugo-clarity/blob/master/CONTRIBUTING.md#how-to-submit-a-pull-request) (using `git commit -s ...`), or if not possible due to developer environment constraints, will comment below confirming that I am adhering to the [Developer Certificate of Origin](https://probot.github.io/apps/dco/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -825,7 +825,18 @@ If you wish use [Utterances](https://github.com/utterance/utterances) comments o
 
 Utterances is loaded in the `comments.html` partial by referring to the `utterances.html` partial.   Since `single.html` layout loads comments if comments are enabled, you must ensure *both* the `comments` and `utterances` parameters are configured.
 
+#### Giscus Commenting Support
 
+If you wish use [giscus](https://giscus.app/) comments on your site, you'll need to perform the following:
```
Comment (chipzoller) [inline README.md:830]:
```suggestion
If you wish to use [giscus](https://giscus.app/) comments on your site, you'll need to perform the following:
```

**Review by chipzoller — COMMENTED** [review]
Thanks for this one too. I have not tested Giscus before, have you tested these changes yourself?

**Comment by chipzoller** [comment]
Please also sign off on your commits (DCO) so we can accept this.

**Comment by rterakedis** [comment]
Hey @chipzoller - It looks like my commit was signed off:

image

Looking at the DCO check, it's actually your update to the readme that isn't signed off?  

image

**Comment by rterakedis** [comment]
> Thanks for this one too. I have not tested Giscus before, have you tested these changes yourself?

✅  I have tested this personally in my own website.  I switched from Utterances to Giscus awhile back and wanted to contribute the code back for others to use (and so I can stop using partial overrides).  😄

**Comment by rterakedis** [comment]
> Thanks for this one too. I have not tested Giscus before, have you tested these changes yourself?

✅  I have tested this personally in my own website.  I switched from Utterances to Giscus awhile back and wanted to contribute the code back for others to use (and so I can stop using partial overrides).  😄

**Review by onweru — APPROVED** [review]

**Pull request merged**